ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -161,11 +164,15 @@ public struct Avatar: View, TokenizedControlView {
 
         let avatarImageInfo: (image: UIImage?, renderingMode: Image.TemplateRenderingMode) = {
             if shouldUseDefaultImage {
-                let isOutlinedStyle = style == .outlined || style == .outlinedPrimary
-                return (UIImage.staticImageNamed(isOutlinedStyle ? "person_48_regular" : "person_48_filled"), .template)
+                var defaultImage = state.defaultImage
+                if defaultImage == nil {
+                    let isOutlinedStyle = style == .outlined || style == .outlinedPrimary
+                    defaultImage = UIImage.staticImageNamed(isOutlinedStyle ? "person_48_regular" : "person_48_filled")
```

Comment:
minor: Should these image names be move to a constants struct?

Reply:
That would be nice! Not a huge fan of having them inline either.